README.md

```diff
@@ -40,3 +40,41 @@ docker pull ghcr.io/input-output-hk/partner-chains/partner-chains-node:latest
 ## How to Use
 
 Refer to the [documentation](docs/user-guides) for detailed instructions on how to use the node and CLI tools.
+
+## Block Production Rewards for Validators
+
+The Partner Chains node provides a simple mechanism for exposing the mapping of block beneficiaries with produced blocks. 
+The Partner Chain Builder is responsible for using this input to accurately calculate and distribute block rewards on their partner chain, following the tokenomics they will design and implement. 
+For managing payouts, they will leverage the provided artifacts to distribute payments to validators on the partner chain ledger, applying their specific business logic.
+The process described below allows for a flexible and fair distribution of rewards, taking into account both block production and stake delegation, while ensuring that only registered participants receive rewards.
+
+| Component | Description |
+|-----------|-------------|
+| Registration | Validators must register their Cardano stake keys with Partner Chain addresses |
```

Comment:
Validators register their Cardano cold secret key with their PC keys (by producing 2 signatures and providing the public keys of each (cardano + PC) so others can verify that the signature matches the pub key)